src/script/interpreter.h

```diff
@@ -185,6 +185,12 @@ enum : uint32_t {
     // Validating OP_CHECKSIGFROMSTACK
     SCRIPT_VERIFY_CHECKSIGFROMSTACK = (1U << 31),
 
+    // Making OP_PAIRCOMMIT non-standard
+    SCRIPT_VERIFY_DISCOURAGE_PAIRCOMMIT = (1U << 30),
+
+    // Validating OP_PAIRCOMMIT
+    SCRIPT_VERIFY_PAIRCOMMIT = (1U << 31),
```

Comment:
These values are already used. Probably better to just reuse one of the variables

Reply:
Since they are all the same anyhow... it's supposed to be a package deal.
```c++
    // Executing OP_INTERNALKEY
    SCRIPT_VERIFY_INTERNALKEY = (1U << 31),

    // Making OP_INTERNALKEY non-standard
    SCRIPT_VERIFY_DISCOURAGE_INTERNALKEY = (1U << 30),

    // Validating OP_CHECKSIGFROMSTACK
    SCRIPT_VERIFY_CHECKSIGFROMSTACK = (1U << 31),

    // Making OP_CHECKSIGFROMSTACK non-standard
    SCRIPT_VERIFY_DISCOURAGE_CHECKSIGFROMSTACK = (1U << 30),

    // Making OP_PAIRCOMMIT non-standard
    SCRIPT_VERIFY_DISCOURAGE_PAIRCOMMIT = (1U << 30),

    // Validating OP_PAIRCOMMIT
    SCRIPT_VERIFY_PAIRCOMMIT = (1U << 31),
```

Reply:
I could simplify this to be the same as in the lnhance repo, single script flags for all LNHANCE opcodes.

Just tried to be consistent with the logic of the codebase I'm PR-ing into. No hard preference either way.

Reply:
ah okay, i guess i did that earlier, thats fine then